client/a8c-for-agencies/sections/marketplace/products-overview/product-filter/hooks/use-product-filter-options.ts

```diff
@@ -27,35 +27,35 @@ export default function useProductFilterOptions() {
 
 	return {
 		[ PRODUCT_FILTER_KEY_CATEGORIES ]: [
-			{ key: PRODUCT_CATEGORY_SECURITY, label: translate( 'Security' ) },
```

Comment:
to fix weird type errors:
<img width="1648" alt="Screenshot 2024-07-17 at 15 29 01" src="https://github.com/user-attachments/assets/8bca39cf-587c-43ea-83b1-e6073478b2a0">

Maybe it's due to some package updates? We might want to look into it later.

Reply:
Looks like trunk is already failing unit test... 😢